01week/datatypes.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+// 1. program for date and time
+function returnDate() {
+  const now = new Date();
+  return now;
+}
+console.log(returnDate());
+
+// 2. convert a number to a string
+const num = 1578;
+const n = num.toString();
+console.log(num);
+
+// 3. convert a string to the number
+const aString = "Mynameisryan";
+console.log(aString.length);
+
+// 4. data type
+function printoutDataType(argument)  {
+  return typeof argument;
+}
+console.log(printoutDataType(5));
+
+// 5. program that adds 2 numbers together
+const x = 10+5;
```

Comment:
This needs to be in a function. A function that accepts two parameters and returns the sum, similar to your "type of" function.